### ISSUE TITLE
HTML 5 Websocket not working correctly

### ISSUE THREAD

**Issue by Sal7one** [kiwibrowser/src#196]
I use Blackboard Collaborate for college and in the latest version it doesn't work correctly.

Live streams just doesn't work. 

It's a shame cause I'd love to use my extensions in my phone.

**Comment by Delphox** [comment]
If you can, try to connect your phone to your PC for debugging ([instructions here](https://developers.google.com/web/tools/chrome-devtools/remote-debugging)) to see what the console says, would be helpful since we don't have access to Blackboard. Also it could be cuz wasm comes disabled by default which causes issues on some sites (see recent issue #197) so that's worth trying too.

**Comment by Sal7one** [comment]
> If you can, try to connect your phone to your PC for debugging ([instructions here](https://developers.google.com/web/tools/chrome-devtools/remote-debugging)) to see what the console says, would be helpful since we don't have access to Blackboard. Also it could be cuz wasm comes disabled by default which causes issues on some sites (see recent issue #197) so that's worth trying too.

Based on my research of which technology they used, This honestly might be it. I will report back in 16 hours when I can access a lecture

Thanks so much

**Comment by Sal7one** [comment]
Alright I tried multiple things none worked.

I had weird warning on mobile console I didn't see in the PC "Invalid asm.js: Expected numeric literal. "

regarding the errors in the live stream page there was one related to WEBRTC

"zsdk_wrtc.js:4 hangupWithError:error_code=-24 reason=Ice connection not established with MCU"

I tried enabling every option related to WEBRTC and Web assembly. I also tried Disabling all of them. None of them worked.

Maybe it's worth mentioning that the blackboard itself recommends FireFox for their live streams
